buddyboss/buddyboss-platform
# PROD-814  PROD-761 Fix - Activity image UI >> equal or portrait images are not appearing properly in the mobile

### All Submissions:


Fixes #2802
Fixes #2435

### How to test the changes in this Pull Request:

1. Go to Activity Feed and upload a similar or portrait picture (4 pictures)
2. Open activity page in the mobile
3. You can see the layout now

### Proof Screenshots or Video
https://prnt.sc/1qlgrgr

### Other information:
